addons/l10n_ma/views/res_partner_views.xml

```diff
@@ -5,13 +5,13 @@
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="account.view_partner_property_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='company_registry']" position="attributes">
-                <attribute name="nolabel">1</attribute>
+            <xpath expr="//field[@name='vat']" position="after">
+                <field name="company_registry" string="ICE" invisible="country_code != 'MA'"></field>
+            </xpath>
+            <xpath expr="//span[hasclass('o_company_id_label')]" position="replace">
```

Comment:
As discussed, we need to get rid of the replace here. Also, Company registry does not match the UI label of the field ;  and the warning for the Tax Id says "Tax Id" instead of using the localized label :/ => We need to fix all that (but in additional commits please ^^)